functions/allFunctions.php

```diff
@@ -0,0 +1,3 @@
+<?php
+include 'connectToDB.php';
+include 'fetchPokemonData.php';
```

Comment:
Use require_once for these, to avoid issues where you might accidentally require the same file twice